MasterSeed32.md

```diff
@@ -274,9 +309,32 @@ Generation of long shares and recovery of the master seed from long shares proce
 The long checksum is designed to be an error correcting code that can correct up to 4 character substitutions, up to 8 unreadable characters (called erasures), or up to 15 consecutive erasures.
 As with regular checksums we do not specify how an implementation should implement error correction, and all our recommendations for error correction of regular MS32 strings also apply to long MS32 strings.
 
-## Test Vectors
+# Rationale
+
+Storing master secrets is the most important component of self-custody.
+Users who do not wish to use multisignature setups or sweep their coins must do so by encoding the master secret and physically encoding it.
+The current best practices for this are either to:
+
+* Use BIP39, which suffers from a weak checksum, uses English words which are have inconsistent lengths and near collisions (or other word lists, the choice of which affects the derived secret but is not explicitly encoded anywhere), and has no natural way to add Shamir Secret Sharing; or
+* Use SLIP39, which also uses English words, includes features such as passphrase-based hardening that may not be worth the additional implementation complexity, and does not have a BIP number.
+
+codex32 is a simpler version of SLIP39 which has a slightly stronger checksum, a more compact and language-independent encoding, and is simple enough that all parts of the scheme (including error detection but not error correction), can be implemented without the use of electronic computers.
+
+# Backwards Compatibility
+
+codex32 is an alternative to BIP39 and SLIP39.
+It is technically possible, but not supported by this BIP, for users to re-encode their BIP39 or SLIP39 secrets in the codex32 format, and to re-encode back.
+However, the only supported way for users to move between codex32 and other formats is to generate a new seed and to sweep their coins.
```

Comment:
I think this needs maybe a bit of work to clarify.

1. It is possible to move master seeds from BIP39 and SLIP39 to Codex32.
2. It is not possible to move shares from SLIP39 to Codex32.
3. It is not possible to move master seeds from Codex32 to BIP39.
4. It is not recommended to move master seeds from BIP39 to Codex32 as it requires storing 512-bit shares and secrets.